src/backend/users/views.py

```diff
@@ -1,46 +1,60 @@
 from django.contrib.auth.models import User
 from django.core.mail import send_mail
+from django.db.utils import IntegrityError
 from django.http import HttpResponseRedirect
 from django.urls import reverse
 from rest_framework.authtoken.models import Token
+from rest_framework.exceptions import ValidationError
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from sesame import utils
 
-from .serializers import UserSerializer
+from .serializers import HouseSeekerSignUpSerializer, UserSerializer
+
+
+def send_login_link(request):
+    """
+    Send email with passwordless login link to User on Login/Sign Up
+    """
+    try:
+        # Request.data key could be "username" or "email", so turn into list to be safe
```

Comment:
Is this in reference to the TODO below or the comment on line 21? I added the comment to add some clarification behind how we get `email` from the `request.data`, but I'm realizing that may be more confusing/unnecessary than helpful! I'll take it out, if so.

Reply:
Ugh, this was a TODO for myself to look into this closer, which I lost track of before submitting the review. Sorry! Anyway, this piqued my interest because it made me wonder whether the request payload could be different depending on whether this was a signup or a signin situation, and whether maybe we should find a way to unify them to make sure it's always consistent. But I didn't actually check into any of those things, so I don't know whether any of those concerns are valid.